### PR TITLE
Replace leftover templates in CODEOWNERS [skip ci]

### PR DESCRIPTION
**Pull request by jakirkham**
cc @ajschmidt8 @quasiben

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,13 +1,13 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
```
Comment (jakirkham) [inline .github/CODEOWNERS:2]: Would it be possible to add @gigony and myself to this team?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-python/            @rapidsai/<repo>-python-codeowners
-python/dask_cudf/  @rapidsai/<repo>-dask-codeowners
+python/            @rapidsai/cucim-python-codeowners
+python/dask_cudf/  @rapidsai/cucim-dask-codeowners
 
 #cmake code owners
-**/CMakeLists.txt  @rapidsai/<repo>-cmake-codeowners
-**/cmake/          @rapidsai/<repo>-cmake-codeowners
+**/CMakeLists.txt  @rapidsai/cucim-cmake-codeowners
+**/cmake/          @rapidsai/cucim-cmake-codeowners
```
Comment (jakirkham) [inline .github/CODEOWNERS:10]: Would it be possible to add @gigony and myself to this team?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,13 +1,13 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
 
 #python code owners
-python/            @rapidsai/<repo>-python-codeowners
-python/dask_cudf/  @rapidsai/<repo>-dask-codeowners
+python/            @rapidsai/cucim-python-codeowners
```
Comment (jakirkham) [inline .github/CODEOWNERS:5]: Would it be possible to add @quasiben, @grlee77, and myself to this team?

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+python/dask_cudf/  @rapidsai/cucim-dask-codeowners
```
Comment (jakirkham) [inline .github/CODEOWNERS:6]: Would it be possible to add @quasiben and myself to this team?

**Review by jakirkham — COMMENTED** [review]
Had a couple questions about adding people to teams below

File: .github/CODEOWNERS (hunk 4 above)
Comment (quasiben) [reply .github/CODEOWNERS:6]: Do we need a cucim-dask-codeowners ?

Comment (jakirkham) [reply .github/CODEOWNERS:6]: Good point. We can drop this. We may want it later, but can always revisit

Comment (jakirkham) [reply .github/CODEOWNERS:6]:
```suggestion
```

File: .github/CODEOWNERS (hunk 2 above)
Comment (ajschmidt8) [reply .github/CODEOWNERS:9]: done: https://github.com/orgs/rapidsai/teams/cucim-cmake-codeowners

File: .github/CODEOWNERS (hunk 1 above)
Comment (ajschmidt8) [reply .github/CODEOWNERS:2]: done: https://github.com/orgs/rapidsai/teams/cucim-cpp-codeowners

File: .github/CODEOWNERS (hunk 3 above)
Comment (ajschmidt8) [reply .github/CODEOWNERS:5]: done: https://github.com/orgs/rapidsai/teams/cucim-python-codeowners

**Review by ajschmidt8 — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks AJ! 😄

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -1,13 +1,12 @@
 #cpp code owners
-cpp/               @rapidsai/<repo>-cpp-codeowners
+cpp/               @rapidsai/cucim-cpp-codeowners
```
Comment (gigony) [inline .github/CODEOWNERS:2]: @jakirkham Could you please add owner for `python/pybind11` folder?

```suggestion
cpp/               @rapidsai/cucim-cpp-codeowners
python/pybind11    @rapidsai/cucim-cpp-codeowners
```

**Review by gigony — APPROVED** [review]
I don't know the good practices on this file (I need to learn more about that) but looks good to me.

**Comment by jakirkham** [comment]
@gpucibot merge

**Pull request merged**